File: dotnet/src/SimpleOrm/SortOrder.cs
namespace SimpleOrm;

/// <summary>
/// Sort direction token for <see cref="IndexAttribute"/> columns: follows the column
/// it applies to, e.g. <c>[Index(nameof(Status), nameof(CreatedAtUtc), SortOrder.Desc)]</c>.
/// A column with no token is ascending.
/// </summary>
public enum SortOrder
{
    Asc,
    Desc,
}

File: php/src/Query/SortOrder.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Query;

/** Ordering direction: an ORDER BY term's, and an `#[Index]` token (ADR-0007 add.3). */
enum SortOrder: string
{
    case Asc = 'asc';
    case Desc = 'desc';
}
